cli/dts/lib.deno.unstable.d.ts

```diff
@@ -1214,4 +1214,46 @@ declare namespace Deno {
    * The pid of the current process's parent.
    */
   export const ppid: number;
+
+  /** **UNSTABLE**: New API, yet to be vetted.
+   * A custom HttpClient for use with `fetch`.
+   * 
+   * ```ts
+   * const client = new Deno.createHttpClient({ caFile: "./ca.pem" });
+   * const req = await fetch("https://myserver.com", { client });
+   * ```
+   */
+  export class HttpClient {
+    rid: number;
+    constructor(rid: number);
+    close(): void;
```

Comment:
Why not use the constructor here for instantiating  a new HttpClient rather than having a different method `createHttpClient` ?

Reply:
https://github.com/denoland/deno/pull/6918#discussion_r463037522